migrate_test.go

```diff
@@ -70,7 +70,7 @@ func ExampleNew() {
 	}
 
 	// Migrate all the way up ...
-	if err := m.Up(); err != nil {
+	if err := m.Up(); err != migrate.ErrNoChange {
```

Comment:
Same issue, don't need to specify the `migrate` package.
Also, you should make sure the `err != nil` as well.